io.github.chkr1011.mqttMultimeter.yaml

```diff
@@ -0,0 +1,37 @@
+app-id: io.github.chkr1011.mqttMultimeter
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+
+command: mqttMultimeter
+
+modules:
+  - name: mqttMultimeter
+    buildsystem: simple
+    build-commands:
+      - unzip mqttMultimeter-linux-x64.zip
+      - chmod +x mqttMultimeter
+      - install -D mqttMultimeter /app/bin/mqttMultimeter
+      - install -D io.github.chkr1011.mqttMultimeter.png /app/share/icons/hicolor/256x256/apps/io.github.chkr1011.mqttMultimeter.png
+      - install -D io.github.chkr1011.mqttMultimeter.desktop /app/share/applications/io.github.chkr1011.mqttMultimeter.desktop
+      - install -D io.github.chkr1011.mqttMultimeter.metainfo.xml /app/share/metainfo/io.github.chkr1011.mqttMultimeter.metainfo.xml
+
+    sources:
+    - type: file
+      url: https://github.com/chkr1011/mqttMultimeter/releases/download/v1.9.0.290/mqttMultimeter-linux-x64.zip
+      sha256: fe7866d67271cda84918b53019aa13ac33fa4b24c870a1a2d17559ed34e8645b
+    - type: file
+      path: io.github.chkr1011.mqttMultimeter.png
+    - type: file
+      path: io.github.chkr1011.mqttMultimeter.desktop
+    - type: file
+      path: io.github.chkr1011.mqttMultimeter.metainfo.xml
+
+finish-args:
+    - --share=ipc
+    - --socket=fallback-x11
+    - --socket=wayland
+    - --device=dri
+    - --share=network
+    - --filesystem=home
```

Comment:
why does it need home access?

Reply:
Because you can save payloads and /or load them before sending. So, we need access to several directories like Documents or Downloads or Desktop etc. Is there a better permission for such a situation?